WheelOfFortune/Prompt.cs

```diff
@@ -151,7 +173,24 @@ public static string PromptInValidEmptyInput()
         public static void GameOverMessage(Player[] winners)
         {
 
+            if (winners.Length == 1)
+            {
+                Console.Write($"Player {winners[0].Name} won the Game.");
+            }
+            else
+            {
+                string winnersNames = "";
+                foreach (Player p in winners)
+                {
+                    winnersNames += "," + p.Name;
```

Comment:
I believe this might create a string that displays as: ",NameOne,NameTwo,NameThree".  Maybe do a regular for loop concatenating the name, and do an ifelse block checking if its the last player put a period, else put a comma and a space